xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -1388,7 +1393,12 @@ void CPeripheralCecAdapter::SetConfigurationFromSettings(void)
 
   // read the mutually exclusive boolean settings
   int iStandbyAction(GetSettingInt("standby_pc_on_tv_standby"));
-  m_configuration.bPowerOffOnStandby = iStandbyAction == LOCALISED_ID_SUSPEND ? 1 : 0;
+
+  if (iStandbyAction == LOCALISED_ID_SUSPEND || iStandbyAction == LOCALISED_ID_HIBERNATE)
+    m_configuration.bPowerOffOnStandby = 1;
+  else
+    m_configuration.bPowerOffOnStandby = 0;
+
```

Comment:
```C
m_configuration.bPowerOffOnStandby = (iStandbyAction == LOCALISED_ID_SUSPEND || iStandbyAction == LOCALISED_ID_HIBERNATE) ? 1 : 0;
```
rather than 4 lines for a single assignment like this please